### DISCUSSION TITLE
🚀 Add common 'Toggle' field to Bulk Edit tool

### DISCUSSION THREAD

**Discussion by ic360-com** [ExpressionEngine/ExpressionEngine#4240] [Feature Requests]
### Suggest an idea for ExpressionEngine

Where a 'Toggle' field is a common field amongst selected Entries, it would really helpful if this was available within the Bulk Edit tool, for setting in a single batch.

**Comment by intoeetive** [comment]
Might get it with https://github.com/ExpressionEngine/ExpressionEngine/pull/4273 (in progress now)
